### DISCUSSION TITLE
Fourth order IFCS from Alamode

### DISCUSSION THREAD

**Discussion by Xuntao1** [ttadano/alamode#152] [General]
Dear users and developers,

Is it possible to extract the 4th oRder ifCS from Alamode in ShengBTE format.

Thanks and Best,
Xun

**Comment by triphysics** [comment]
Hi Xun,

This  I think has been implemented. Please check the following previous discussion:

https://github.com/ttadano/alamode/discussions/93

Best,
Trib

**Comment by Xuntao1** [comment]
Thanks for the prompt response. This is very helpful.

Xun
